buildSrc/src/main/kotlin/openapi-conventions.gradle.kts

```
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.gradle.api.tasks.compile.JavaCompile
+import org.gradle.kotlin.dsl.extra
+import org.gradle.kotlin.dsl.withType
+import org.openapitools.generator.gradle.plugin.extensions.OpenApiGeneratorGenerateExtension
+
+plugins {
+    id("org.openapi.generator")
+}
+
+// Export the function as an extra property
+val configureOpenApi by
+extra(
+    fun(extension: OpenApiGeneratorGenerateExtension) {
+        val openApiPackage = "com.hedera.mirror.rest"
+        extension.apiPackage = "${openApiPackage}.api"
```

We should no longer export this to extra.

```suggestion
val openApiPackage = "com.hedera.mirror.rest"
openApiGenerate {
    apiPackage = "${openApiPackage}.api"
    ...
```

---

Done